collision_clearance: 0.15

collision_links:
  r_gripper_l_finger_link:
    link_radius: 0.04
    link_extension: 0.03
  r_gripper_r_finger_link:
    link_radius: 0.04
    link_extension: 0.03
  r_gripper_palm_link:
    link_radius: 0.06
  r_forearm_link:
    link_radius: 0.065
  r_upper_arm_link:
    link_radius: 0.07

joint_costs:
  r_shoulder_pan_joint: 1.0
  r_shoulder_lift_joint: 1.0
  r_upper_arm_roll_joint: 1.0
  r_elbow_flex_joint: 1.0
  r_forearm_roll_joint: 1.0
  r_wrist_flex_joint: 1.0
  r_wrist_roll_joint: 1.0

joint_velocity_limits:
  r_shoulder_pan_joint: 0.5
  r_shoulder_lift_joint: 0.5
  r_upper_arm_roll_joint: 0.7
  r_elbow_flex_joint: 0.8
  r_forearm_roll_joint: 1.0
  r_wrist_flex_joint: 1.0
  r_wrist_roll_joint: 1.0


reference_frame: base_link

collision_space:
  size_x: 1.2
  size_y: 2.0
  size_z: 2.2
  origin_x: -0.1
  origin_y: -1.5
  origin_z: -0.3
  resolution: 0.01
  field_bias_x: 0.0
  field_bias_y: 0.0
  field_bias_z: 0.0
  cuboids:
    cuboid1:
      size_x: 0.6
      size_y: 0.6
      size_z: 0.3
      origin_x: -0.3
      origin_y: -0.3
      origin_z: 0.0
      
